package.json

```diff
@@ -123,6 +123,7 @@
     "chromatic": "^6.15.0",
     "concurrently": "^7.6.0",
     "cross-env": "^7.0.3",
+    "d3-scale": "^4.0.2",
```

Comment:
as it stands this upgrade will break our consumers because v4 moves to ESM only.

Can you use d3-scale from `victory-vendor/d3-scale` instead?